py-polars/polars/expr/expr.py

```diff
@@ -204,7 +207,10 @@ def __ror__(self, other: Any) -> Self:
         return self._from_pyexpr(self._to_pyexpr(other)._or(self._pyexpr))
 
     def __pos__(self) -> Expr:
-        return F.lit(0) + self
+        pos_expr = F.lit(0) + self
```

Comment:
Why do we actually need to do this operation, instead of returning self?

Reply:
I think because it will catch incorrect usage in the way that a no-op won't; it's an implied numeric operation, but if we just returned `self` we'd miss that (for instance: `+pl.col("some_string_col")` _should_ raise an error - which the current implementation will do, but a no-op wouldn't).